src/static/bitbucket/components/App.tsx

```diff
@@ -12,14 +18,32 @@ export type AppState = {
   canLand: boolean;
   canLandWhenAble: boolean;
   errors: string[];
+  bannerMessage: BannerMessage;
 };
 
 export class App extends React.Component {
+  messageColours = {
+    default: 'transparent',
+    warning: 'orange',
+    error: 'red',
+  };
+
+  messageEmoji = {
+    default: '📢',
+    warning: '⚠️',
+    error: '❌',
```

Comment:
There is an addition space after the X

Reply:
maybe it is expected

Reply:
There actually isn't, it is misleading though, just how the emoji renders.